engine/wwtlib/ScriptInterface.cs

```diff
@@ -446,6 +446,32 @@ private static ImageSetLayer AddFitsLayer(string url, string name, bool gotoTarg
             return imagesetLayer;
         }
 
+        public void SetImageSetLayerOrder(string id, int order)
+        {
+            foreach (string referenceFrame in LayerManager.AllMaps.Keys)
+            {
+                Layer layerToMode = GetLayerFromReferenceFrame(id, referenceFrame);
+                
+                if(order >= 0 && layerToMode != null)
+                {
+                    LayerManager.AllMaps[referenceFrame].Layers.Remove(layerToMode);
+                    LayerManager.AllMaps[referenceFrame].Layers.Insert(order, layerToMode);
```

Comment:
What happens if `order` is bigger than `LayerManager.AllMaps[referenceFrame].Layers.Length`?

Reply:
They end up at the end of the list - (It is converted to a javascript array splice operation)

I added an explanatory comment for any future person having the same reasonable question.